src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -556,28 +585,33 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
 
             case NI_Vector64_Create:
             case NI_Vector128_Create:
-                if (intrin.op1->isContainedFltOrDblImmed())
-                {
-                    const double dataValue = intrin.op1->AsDblCon()->gtDconVal;
-                    GetEmitter()->emitIns_R_F(INS_fmov, emitSize, targetReg, dataValue, opt);
-                }
-                else if (varTypeIsFloating(intrin.baseType))
-                {
-                    GetEmitter()->emitIns_R_R_I(ins, emitSize, targetReg, op1Reg, 0, opt);
-                }
-                else
+            case NI_AdvSimd_DuplicateToVector64:
+            case NI_AdvSimd_DuplicateToVector128:
+            case NI_AdvSimd_Arm64_DuplicateToVector128:
+            {
+                if (varTypeIsFloating(intrin.baseType))
                 {
-                    if (intrin.op1->isContainedIntOrIImmed())
+                    if (intrin.op1->isContainedFltOrDblImmed())
                     {
-                        const ssize_t dataValue = intrin.op1->AsIntCon()->gtIconVal;
-                        GetEmitter()->emitIns_R_I(INS_movi, emitSize, targetReg, dataValue, opt);
+                        const double dataValue = intrin.op1->AsDblCon()->gtDconVal;
+                        GetEmitter()->emitIns_R_F(INS_fmov, emitSize, targetReg, dataValue, opt);
                     }
                     else
                     {
-                        GetEmitter()->emitIns_R_R(ins, emitSize, targetReg, op1Reg, opt);
+                        GetEmitter()->emitIns_R_R_I(ins, emitSize, targetReg, op1Reg, 0, opt);
                     }
                 }
+                else if (intrin.op1->isContainedIntOrIImmed())
+                {
+                    const ssize_t dataValue = intrin.op1->AsIntCon()->gtIconVal;
+                    GetEmitter()->emitIns_R_I(INS_movi, emitSize, targetReg, dataValue, opt);
+                }
+                else
+                {
+                    GetEmitter()->emitIns_R_R(ins, emitSize, targetReg, op1Reg, opt);
+                }
                 break;
```

Comment:
>break; [](start = 16, length = 6)

minor: for consistency, can you put the `break` outside the parenthesis of case?

Reply:
Fixed.

Is this a convention just for the ARM code? I've not seen it in other parts of the codebase...

Reply:
I have seen at few places in this file so that's why thought of keeping it consistent at least in this file. :)